### PR TITLE
Add Header and ProductOverview tests

### PR DESCRIPTION
**Pull request by mattsaxe17**

### CODE REVIEW

File: client/src/components/ProductOverview/ProductExtra.jsx [hunk 1]
```diff
@@ -2,34 +2,22 @@ import React from 'react';
...
 
 const ProductExtra = (props) => (
-  <div>
+  <div id="product-extra">
     <div id="product-description">
-      <h4 id="description-title">Product slogan or catchphrase...</h4>
-      <p id="description">Spicy jalapeno bacon ipsum dolor amet burgdoggen pork belly esse bacon, bresaola strip steak ut drumstick eiusmod chicken flank ea sed. Laboris corned beef aliqua sausage pancetta ball tip rump bacon qui spare ribs nostrud. Hamburger beef incididunt, cow fugiat do prosciutto pastrami filet mignon pancetta deserunt lorem sausage <br /><br />andouille picanha. Ham cupidatat ipsum, spare ribs esse velit kielbasa magna in doner cupim. Dolore reprehenderit adipisicing ullamco.
-
-        Short loin dolore cow laborum culpa velit nostrud irure.  Qui pork loin nisi bresaola cillum anim pig salami dolore nostrud.</p>
+      <h4 id="description-title">{props.product?.slogan}</h4>
```
Comment (LesliePajuelo) [inline client/src/components/ProductOverview/ProductExtra.jsx:7]: What is that? why is there a question mark as a key name? that's odd.

File: client/src/components/ProductOverview/ProductOverview.scss [hunk 2]
```diff
@@ -243,13 +243,14 @@ $standardBorder: 1px solid $standardColor;
     display: flex;
     justify-content: center;
 
+    #product-description {
+      border-right: $standardBorder;
+      border-width: 2px;
```
Comment (LesliePajuelo) [inline client/src/components/ProductOverview/ProductOverview.scss:248]: Consider not using a hard coded pixel, remember Leslie said about hard coding, maybe 0.5 ems?

**Review by LesliePajuelo — APPROVED** [review]
great job!

File: client/src/components/ProductOverview/ProductDetails.jsx [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { AiOutlineCheck, AiOutlineStar } from 'react-icons/ai';
 import Stars from '../Stars/Stars.jsx';
 import CustomSelect from './CustomSelect.jsx';
 import helpers from '../../helpers.js';
+import _ from 'lodash';
```
Comment (cmac0351) [inline client/src/components/ProductOverview/ProductDetails.jsx:6]: maybe we could only import the methods we are using from lodash

File: client/src/components/ProductOverview/ProductExtra.jsx [hunk 4]
```diff
@@ -2,34 +2,22 @@ import React from 'react';
...
+      <p id="description">{props.product?.description}</p>
```
Comment (cmac0351) [inline client/src/components/ProductOverview/ProductExtra.jsx:8]: I need to learn more about optional chaining

File: client/src/components/ProductOverview/ProductImage.jsx [hunk 5]
```diff
@@ -34,7 +34,7 @@ const ProductImage = (props) => {
 
 
   return (
-    <div id="product-image" className="bg-image" style={{ backgroundImage: `url(${props.images[currentImageIndex]?.url})` }}>
+    <div id="product-image" className="bg-image" style={{ backgroundImage: props.images[currentImageIndex]?.url ? `url(${props.images[currentImageIndex]?.url})` : 'none' }}>
```
Comment (cmac0351) [inline client/src/components/ProductOverview/ProductImage.jsx:37]: nice!, now we won't get that 'undefined' network event

File: client/src/components/ProductOverview/ProductOverview.test.js [hunk 6]
```diff
@@ -1,38 +1,82 @@
...
+    expect(productExtraWrapper.find('#product-description')).toHaveLength(1);
+    expect(productExtraWrapper.find('#product-features')).toHaveLength(1);
+    done();
+  });
+});
 
-  //   return Promise.all([product, productStyles]);
-  // });
+describe('ProductOverview API', () => {
 });
```
Comment (cmac0351) [inline client/src/components/ProductOverview/ProductOverview.test.js:82]: github, and HR style guide likes a blank line as the last line of the module

File: client/src/components/ProductOverview/ProductOverviewAPIUtils.js [hunk 7]
```diff
@@ -31,7 +31,6 @@ export default {
     return new Promise((resolve, reject) => {
       axios.get(`${url}/api/reviews/${id}`)
         .then(response => {
-          console.log(response);
           resolve(response.data);
         })
         .catch(err => {
```
Comment (cmac0351) [inline client/src/components/ProductOverview/ProductOverviewAPIUtils.js:36]: maybe we can convert this axios call to use async/await

File: client/src/components/Reviews/ReviewsAPIUtils.js [hunk 8]
```diff
@@ -34,6 +34,6 @@ export default {
   },
 
   submitFormData: (formData, config) => {
-    return axios.post('api/reviews', formData, config);
+    return axios.post('/api/reviews', formData, config);
```
Comment (cmac0351) [inline client/src/components/Reviews/ReviewsAPIUtils.js:37]: thanks, nice catch!

File: client/src/styles.scss [hunk 9]
```diff
@@ -13,7 +13,10 @@ html {
 body {
   width: 100%;
   min-height: 100vh;
+<<<<<<< HEAD
```
Comment (cmac0351) [inline client/src/styles.scss:16]: looks like there is some text from a merge conflict left over

**Review by cmac0351 — APPROVED** [review]

**Pull request merged**